cluster/up.sh

```diff
@@ -5,10 +5,15 @@ set -ex
 source ./cluster/kubevirtci.sh
 kubevirtci::install
 
+kubectl=./cluster/kubectl.sh
+
 $(kubevirtci::path)/cluster-up/up.sh
 
+# Convert node02 into master
+$kubectl label node node02 node-role.kubernetes.io/master="" node-role.kubernetes.io/worker-
```

Comment:
Why?

Reply:
To check leader election at POC we need two masters.